Winch/Core/AssetLoader.cs

```diff
@@ -33,17 +34,37 @@ private static void LoadAssetFolder(string path)
 
         private static void LoadItemFiles(string itemFolderPath)
         {
-            string spatialItemsPath = Path.Combine(itemFolderPath, "General");
-            string fishItemsPath = Path.Combine(itemFolderPath, "Fish");
+            Dictionary<Type, string> _pathData = new Dictionary<Type, string>()
```

Comment:
This is more succint but I miss the relationship between types here. We have no information of inheritance nor of common handling. In this case we don't need it since we are dealing with data definitions so there is absolutely no problem. 

Just making a comment, no change necessary.